sr_unix_gnp/cmj_getsockaddr.c

```diff
@@ -156,6 +156,7 @@ cmi_status_t cmj_getsockaddr(cmi_descriptor *nod, cmi_descriptor *tnd, struct ad
 		SERVER_HINTS(hints, (ipv4_only ? AF_INET : AF_UNSPEC));
 		if (0 != (errcode = getaddrinfo(NULL, GTCM_SERVER_NAME, &hints, &ai_ptr)))
 		{
+			freeaddrinfo(ai_ptr);
```

Comment:
In this case, getaddrinfo() returned failure. So freeaddrinfo() seems incorrect. What was the original warning?

Reply:
Yes, it got added in the wrong place. Moved up a few lines to around line 152 which is where the original resource leak issue was.